src/parser/shaman/enhancement/modules/talents/ForcefulWinds.js

```diff
@@ -5,12 +5,12 @@ import { formatNumber, formatPercentage } from 'common/format';
 
 import Analyzer from 'parser/core/Analyzer';
 
-import StatisticBox from 'interface/others/StatisticBox';
+import TalentStatisticBox from 'interface/others/TalentStatisticBox';
 import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
 
 
 const FORCEFUL_WINDS = {
-  INCREASE: 1,
+  INCREASE: 0.8,
 };
```

Comment:
Shouldn't this just be a variable and not an object? There seems to be nothing else related to this so just naming it ``const FORCEFUL_WINDS_INCREASE = 0.8`` would be more consistent

Reply:
I agree, it was already like than once i touched the file and didn't consider changing it.